File: src/app/VehicleListDto.ts
export interface VehicleListDto {
  id: number;
  modelId: number;
  modelName?: string;
  brandName?: string;
  brandId: number;
  status?: string;
  ilanda?: boolean;
  yil?: string;
  km?: string;
  yakitTuru?: string;
  imageUrl?:string;
}

File: Models/VehicleListDto.cs
namespace ElvanCarApp.Models
{
    public class VehicleListDto
    {

        public int Id { get; set; }

        public int? ModelId { get; set; }
        public string? ModelName { get; set; } = null!;

        public int? BrandId { get; set; }
        public string? BrandName { get; set; } = null!;

        public string? Status { get; set; }
        public bool? Announcement { get; set; }
        public int? Year { get; set; }
        public int? KM { get; set; }
        public string? FuelType { get; set; }
        public string? ImageUrl { get; set; }
    }
}
